liberty-maven-plugin/src/it/generate-features-it/src/test/java/net/wasdev/wlp/test/dev/it/BaseGenerateFeaturesTest.java

```diff
@@ -67,7 +67,7 @@ public class BaseGenerateFeaturesTest {
     static File serverXmlFile;
 
     static final String GENERATED_FEATURES_FILE_NAME = "generated-features.xml";
-    static final String GENERATED_FEATURES_FILE_PATH = "/src/main/liberty/config/configDropins/overrides/" + GENERATED_FEATURES_FILE_NAME;
+    static final String GENERATED_FEATURES_FILE_PATH = "/target/liberty/wlp/usr/servers/defaultServer/configDropins/overrides/" + GENERATED_FEATURES_FILE_NAME;
```

Comment:
I think it's possible that the path to "configDropins/overrides" may not always be exactly this. I think users can change their "install dir", "output dir", and "server name"... maybe more as well.

If my assumption is true, let's open a new issue to address handling that separately. We can still move forward with this PR as is.

Reply:
My mistake, this is a test file. The various potential custom paths will be covered by the `serverDirectory` variable used on this line: https://github.com/OpenLiberty/ci.maven/pull/1877/files#diff-f13f2e96a8e7ac60edcefdbaf97779e5f6926ac505277f90e8fa1d3b46507384R274.